ghc-mod.cabal

```diff
@@ -201,6 +205,8 @@ Executable ghc-mod
                       , mtl       < 2.3 && >= 2.0
                       , ghc       < 7.11
                       , fclabels  ==2.0.*
+                      , optparse-applicative ==0.11.*
+                      , ansi-wl-pprint ==0.6.*
```

Comment:
Btw, I didn't find time to check, but I think that we may relax constraints on optparse-applicative to include 0.12.* as well. Judging from changelog, it doesn't seem that 0.12 introduces breaking changes.